apps/src/music/player/MusicPlayer.ts

```diff
@@ -16,6 +16,7 @@ import Lab2Registry from '@cdo/apps/lab2/Lab2Registry';
 import {LoadFinishedCallback, UpdateLoadProgressCallback} from '../types';
 import {AudioPlayer, SampleEvent, SamplerSequence} from './types';
 import SamplePlayerWrapper from './SamplePlayerWrapper';
+import {DEFAULT_PATTERN_LENGTH, DEFAULT_CHORD_LENGTH} from '../constants';
```

Comment:
Oh thanks for fixing this! Can we remove the `require` below too?

Reply:
Done.